commit:
https://github.com/OpenPLi/enigma2/commit/cc5162f4cd80e2c23a7379785ea0b02ee28d52b3
triggers the triple tuner configuration bug which we still have in Enigma2.
nimmanager.somethingConnected() returns always False and due to that users have no menu option like Fast Scan,
Automatic scan, Manual scan, Positioner setup and signalfinder.

diff --git a/data/menu.xml b/data/menu.xml
index 5afae97c9..217c3c4a5 100644
--- a/data/menu.xml
+++ b/data/menu.xml
@@ -14,7 +14,7 @@
 		"exec"s code
 -->
 		<item level="0" text="Timer" entryID="timer_edit"><screen module="TimerEdit" screen="TimerEditList"/></item>
-		<item level="1" text="VCR scart" entryID="scart_switch" conditional="config.usage.show_vcr_scart.value" requires="ScartSwitch"><code>self.session.scart.VCRSbChanged(3)</code></item>
+		<item level="1" text="VCR scart" entryID="scart_switch" configcondition="config.usage.show_vcr_scart" requires="ScartSwitch"><code>self.session.scart.VCRSbChanged(3)</code></item>
 
 		<menu level="0" text="Information" entryID="info_screen">
 			<id val="information"/>
@@ -29,8 +29,8 @@
 			<menu weight="7" level="0" text="Service searching" entryID="service_searching_selection">
 				<id val="scan"/>
 				<item text="Tuner configuration" entryID="tuner_setup"><screen module="Satconfig" screen="NimSelection"/></item>
-				<item text="Automatic scan" entryID="auto_scan" conditional="nimmanager.somethingConnected()"><screen module="ScanSetup" screen="ScanSimple"/></item>
-				<item text="Manual scan" entryID="manual_scan" conditional="nimmanager.somethingConnected()"><screen module="ScanSetup"/></item>
+				<item text="Automatic scan" entryID="auto_scan"><screen module="ScanSetup" screen="ScanSimple"/></item>
+				<item text="Manual scan" entryID="manual_scan"><screen module="ScanSetup"/></item>
 				<item text="Fallback remote receiver setup" entryID="fallbacktuner_settings"><screen module="SetupFallbacktuner"/></item>
 			</menu>
 			<menu weight="5" level="0" text="System" entryID="system_selection">
diff --git a/lib/python/Plugins/SystemPlugins/FastScan/plugin.py b/lib/python/Plugins/SystemPlugins/FastScan/plugin.py
index 67bb2db8d..508d11603 100644
--- a/lib/python/Plugins/SystemPlugins/FastScan/plugin.py
+++ b/lib/python/Plugins/SystemPlugins/FastScan/plugin.py
@@ -380,7 +380,7 @@ def autostart(reason, **kwargs):
 		config.misc.standbyCounter.removeNotifier(standbyCountChanged)
 
 def FastScanStart(menuid, **kwargs):
-	if menuid == "scan" and nimmanager.somethingConnected():
+	if menuid == "scan":
 		return [(_("Fast Scan"), FastScanMain, "fastscan", None)]
 	else:
 		return []
diff --git a/lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py b/lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py
index 5cb239e3a..9b3c14a3d 100644
--- a/lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py
+++ b/lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py
@@ -1573,7 +1573,7 @@ def PositionerMain(session, **kwargs):
 			session.open(MessageBox, _("No tuner is configured for use with a diseqc positioner!"), MessageBox.TYPE_ERROR)
 
 def PositionerSetupStart(menuid, **kwargs):
-	if menuid == "scan" and nimmanager.somethingConnected():
+	if menuid == "scan":
 		return [(_("Positioner setup"), PositionerMain, "positioner_setup", None)]
 	else:
 		return []
diff --git a/lib/python/Plugins/SystemPlugins/Satfinder/plugin.py b/lib/python/Plugins/SystemPlugins/Satfinder/plugin.py
index 67ffc7c0d..6ffb10fe1 100644
--- a/lib/python/Plugins/SystemPlugins/Satfinder/plugin.py
+++ b/lib/python/Plugins/SystemPlugins/Satfinder/plugin.py
@@ -549,7 +549,7 @@ def SatfinderMain(session, close=None, **kwargs):
 		session.openWithCallback(boundFunction(SatfinderCallback, close), Satfinder)
 
 def SatfinderStart(menuid, **kwargs):
-	if menuid == "scan" and nimmanager.somethingConnected():
+	if menuid == "scan":
 		return [(_("Signal finder"), SatfinderMain, "satfinder", None)]
 	else:
 		return []
diff --git a/lib/python/Screens/Menu.py b/lib/python/Screens/Menu.py
index 9cbc7dfbc..37702b90b 100644
--- a/lib/python/Screens/Menu.py
+++ b/lib/python/Screens/Menu.py
@@ -6,7 +6,6 @@ from Components.ActionMap import NumberActionMap, ActionMap
 from Components.Sources.StaticText import StaticText
 from Components.config import configfile
 from Components.PluginComponent import plugins
-from Components.NimManager import nimmanager
 from Components.config import config, ConfigDictionarySet, NoSave
 from Components.SystemInfo import SystemInfo
 from Components.Label import Label
@@ -117,8 +116,8 @@ class Menu(Screen, ProtectedScreen):
 					return
 			elif not SystemInfo.get(requires, False):
 				return
-		conditional = node.get("conditional")
-		if conditional and not eval(conditional):
+		configCondition = node.get("configcondition")
+		if configCondition and not eval(configCondition + ".value"):
 			return
 		item_text = node.get("text", "").encode("UTF-8")
 		entryID = node.get("entryID", "undefined")
-- 

